CONTRIBUTING.md

```diff
@@ -55,7 +56,7 @@ There are 3 test suites:
 - **Integration tests** - some tests are flagged as integration as they can take more than a few milliseconds to complete. It's usually recommended to separate them from the rest of the unit tests that run fast. Usually they include disk I/O operations, network I/O operations on a test port, or encryption computations. We also rely on the kubebuilder testing framework, that spins up etcd and the apiserver locally, and enqueues requests to a reconciliation function.
 
 - **End-to-end tests** - (e2e) allow us to test interactions between the operator and a real Kubernetes cluster.
-      They use the standard `go test` tooling. See the `test/e2e` directory. We recommend to rely primarily on unit and integration tests, as e2e tests are slow and hard to debug because they simulate real user scenarios.
+      They use the standard `go test` tooling. See the `test/e2e` directory. We recommend to rely primarily on unit and integration tests, as e2e tests are slow and hard to debug because they simulate real user scenarios. To run a specific e2e test, you can use something similar to `make TESTS_MATCH=TestMetricbeatStackMonitoringRecipe clean docker-build docker-push e2e-docker-build e2e-docker-push e2e-run`. This will run the e2e test with your latest commit.
```

Comment:
I almost always run `make run` in one shell, and `make e2e-local TESTS_MATCH= TestMetricbeatStackMonitoringRecipe` in another shell, maybe also worth mentioning here since it gives a much faster workflow.

Reply:
That hides some of the issues that will happen in the CI (e.g. some setup missing, no permissions, etc.). But +1 on mentioning it.